util/install/sabayon.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+_qmk_install() {
+    echo "Installing dependencies"
+
+    sudo equo install \
+        app-arch/unzip app-arch/zip net-misc/wget sys-devel/clang sys-devel/gcc sys-devel/crossdev \
+        dev-lang/python \
+        dev-embedded/avrdude dev-embedded/dfu-programmer app-mobilephone/dfu-util
+
+    sudo crossdev -s4 --stable --g \<9 --portage --verbose --target avr
+    sudo crossdev -s4 --stable --g \<9 --portage --verbose --target arm-none-eabi
+
+    python3 -m pip install --user -r $QMK_FIRMWARE_DIR/requirements.txt
+}
```

Comment:
After switching to the `sabayonlinux.org` repo (rather than `sabayon-weekly`) I was able to update the distro - for some reason the weekly repo wanted to upgrade mesa to a version that was not in any of the mirrors.
Now, the `crossdev` commands fail with:
```
/usr/bin/crossdev: line 1019: multilib_env: command not found

 * error: could not load multilib settings for HCHOST
```
🤷